src/storeshield/routes/files.py

```
@@ -0,0 +1,136 @@
+import os
+from datetime import datetime, timezone, timedelta
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import

(F401)